_content/reference/buddhist-monasteries-meditation-centresi-sri-lanka.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Buddhist Forest Monasteries and Meditation Centres in Sri Lanka: Guide for Foreign Buddhist Practitioners"
+external_url: "https://www.lotuslibrary.com/ebooks/buddhist-forest-monasteries-and-meditation-centres-in-sri-lanka/"
```

Comment:
This is the old version. Please link to the new version: https://placestomeditate.wordpress.com/2018/08/11/overview-of-sri-lanka-monasteries/